### DISCUSSION TITLE
Single Release View - Add Navigation to View Other Releases

### DISCUSSION THREAD

**Discussion by larsvansoest** [community/community#6119] [Other Feature Feedback, Questions, & Ideas] [answered]
Personally, for the single release view, reading patch notes, I enjoy the focus assist provided by reducing the page's information to the release's data. However, I couldn't help but feel a bit trapped, as I had to go back to the releases overview to view another release.

To enable switching to other releases in single view, I would like to suggest introducing some navigation buttons.

For instance, two buttons such as (previous release | next release) could already be quite helpful. Or, a small menu could be introduced to select which release is shown.

**Comment by MylesBorins** [comment]
@larsvansoest this is definitely an interesting idea. Since this would be a net new feature beyond our current UI changes (which aimed to be mostly feature equivalent) we are going to want to wait to explore this design space until we wrap up our current work. I'll revisit this discussion when we have a bit more bandwidth.

Thanks for the idea!

**Reply by larsvansoest** [reply]
You're welcome!

**Comment by MylesBorins** [comment, marked as answer]
Hey @larsvansoest after a bunch of thought this isn't something we are going to be able to move forward with at this time. Thank you again for taking the time to share your feedback!
